.github/workflows/pullRequests.yaml

```diff
@@ -20,6 +20,7 @@ jobs:
     strategy:
       matrix:
         dir: [".", "example"]
+        os: [ubuntu-latest, windows-latest, macos-latest]
     runs-on: ubuntu-latest
```

Comment:
doesn't this need to run with the matrix variable

Reply:
Good catch, fixed